randovania/data/json_data/prime1.txt

```diff
@@ -408,310 +625,681 @@ Asset id: 2199198515
 Hydra Lab Entryway
 Asset id: 961011783
 > Dock to Research Entrance; Heals? False
-  * Normal Door to Research Entrance/Dock to Hydra Lab Entryway
+  * Wave Door to Research Entrance/Dock to Hydra Lab Entryway
+  > Dock to Research Lab Hydra
+      Trivial
 
 > Dock to Research Lab Hydra; Heals? False
-  * Normal Door to Research Lab Hydra/Dock to Hydra Lab Entryway
+  * Wave Door to Research Lab Hydra/Dock to Hydra Lab Entryway
+  > Dock to Research Entrance
+      Trivial
 
 ----------------
 Quarantine Cave
 Asset id: 1880625556
 > Dock to North Quarantine Tunnel; Heals? False
-  * Normal Door to North Quarantine Tunnel/Dock to Quarantine Cave
+  * Wave Door to North Quarantine Tunnel/Dock to Quarantine Cave
+  > Dock to South Quarantine Tunnel
+      Morph Ball and Spider Ball and After Thardus
+  > Event (Thardus)
+      Trivial
 
 > Dock to South Quarantine Tunnel; Heals? False
-  * Normal Door to South Quarantine Tunnel/Dock to Quarantine Cave
+  * Wave Door to South Quarantine Tunnel/Dock to Quarantine Cave
+  > Dock to North Quarantine Tunnel
+      Morph Ball and Spider Ball and After Thardus
+  > Dock to Quarantine Monitor
+      Grapple Beam and Morph Ball
+  > Event (Thardus)
+      Trivial
 
 > Dock to Quarantine Monitor; Heals? False
   * Normal Door to Quarantine Monitor/Dock to Quarantine Cave
+  > Dock to North Quarantine Tunnel
+      Morph Ball
+  > Dock to South Quarantine Tunnel
+      Morph Ball
+  > Event (Thardus)
+      Trivial
```

Comment:
- Remove connection to North Quarantine Tunnel
- Connection to South Quarantine Tunnel also need Grapple